vllm/transformers_utils/config.py

```diff
@@ -492,6 +492,9 @@ def get_sentence_transformer_tokenizer_config(model: str,
             if encoder_dict:
                 break
 
+    if Path(model).exists():
+        return None
+
     if not encoder_dict:
```

Comment:
@maxdebayser maybe I don't understand but wouldn't you want to still potentially return the encoder_dict in this case?

```suggestion
    if not encoder_dict and not Path(model).exists():
```

Reply:
Yes! What was I thinking?